UITests/UITests.App/App.AppService.xaml.cs

```diff
@@ -139,9 +140,10 @@ public async Task SendLogMessage(string level, string msg)
                 { "Message", msg }
             };
 
-            await _appServiceConnection.SendMessageAsync(message);
+            var response = await _appServiceConnection.SendMessageAsync(message);
 
             // TODO: do we care if we have a problem here?
+            Debug.WriteLine(response);
```

Comment:
We don't need to declare an unused local just to suppress the warning, we have the discard operator.
We can also remove the new `System.Diagnostics` directive at the top for the same reason.

```suggestion
            // TODO: do we care if we have a problem here?
            _ = await _appServiceConnection.SendMessageAsync(message);
```

Reply:
It'll be useful in debug scenarios, where we can see the response data written to the debug console!

Reply:
We currently have the `LogMessage` method do Debug output, so I think that's enough for now. If we hit an issue in this area, we can always add something locally to help debug (if we can't inspect the return value from the function, VS does keep that sometimes in the local window).

Reply:
I added this in response to that comment. No special reason!

Reply:
Ah, I see. Yeah, I had added that comment when I first wrote it as I wasn't sure what to expect. It's definitely more of a rhetorical/hypothetical comment. In the short-term, I don't think we need to worry about it as we do have the other outputs.

@Sergio0694 we could just not `await` here and return using `AsTask` on the IAsyncOperation. Just pass the context up and let the caller of this helper decide what to do in case of a problem, eh?